cassandra-analytics-integration-tests/src/test/java/org/apache/cassandra/analytics/SparkBulkAnalyticsSimpleTest.java

```diff
@@ -40,7 +40,7 @@
 import static org.apache.cassandra.testing.TestUtils.ROW_COUNT;
 import static org.apache.cassandra.testing.TestUtils.TEST_KEYSPACE;
 
-class SparkBulkWriterSimpleTest extends SharedClusterSparkIntegrationTestBase
+class SparkBulkAnalyticsSimpleTest extends SharedClusterSparkIntegrationTestBase
```

Comment:
Since you are renaming the class.. would it be more fit to name it `CassandraAnalyticsSimpleTest`? 
"SparkBulkAnalytics" means "CassandraAnalytics" here.

Reply:
That was my first choice, so let me change it to `CassandraAnalyticsSimpleTest`